README.md

```diff
@@ -167,7 +167,11 @@ sudo sed -i 's/.*autospawn.*/autospawn = no/g' /etc/pulse/client.conf
 
 sudo sed -i 's/.*autospawn.*/autospawn = no/g' /etc/pulse/client.conf.d/00-enable-autospawn.conf       
 
-# Also If `/etc/xdg/autostart/pulseaudio.desktp` file exist, you have to backup this file to somewhere or have to delete it.
+# Few others Ubuntu flavored distributions may have autospawn defined at some different places, check /etc/init.d/pulseaudio-enable-autospawn
+
+sudo update-rc.d pulseaudio-enable-autospawn remove
```

Comment:
@avoidik 

This will throw an error as `pulseaudio-enable-autospawn` script still present under `/etc/init.d/` until use `-f` option (this will remove the script also from `/etc/init/`)

Although why `remove`? AFAIK this will disable the service until next time the service is upgraded. instead you can use `disable` option which is make the service disabled until you manually enable it?

Reply:
changed to disable

Reply:
@avoidik Can you also change 
> \# Few others Ubuntu flavored distributions may have ...

to

> \# PulseAudio (>= 13.04-4) have ...

Because the original `pulseaudio` (>= 13.0-4) maintainer included this init script under `debian/` dir and all `debian/ubuntu` flavoured package maintainer following the original one, So under every distro this script is included (Except Mx Linux, PA  < 13.04-4). 

I'm asking you so that a clean merge can be performed after this little change